actions/docs-verifier/src/MarkdownLinksVerifier/LinkClassifier/Classifier.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace MarkdownLinksVerifier.LinkClassifier
+{
+    internal static class Classifier
+    {
+        public static LinkClassification Classify(string link)
+        {
+            if (link.StartsWith("https://", StringComparison.OrdinalIgnoreCase) ||
+                link.StartsWith("http://", StringComparison.OrdinalIgnoreCase) ||
+                link.StartsWith("ftp://", StringComparison.OrdinalIgnoreCase))
```

Comment:
```suggestion
            if (link.StartsWith("http", StringComparison.OrdinalIgnoreCase) ||
                link.StartsWith("ftp", StringComparison.OrdinalIgnoreCase))
```

Reply:
@IEvangelist This can yield to incorrect results, for example from docs: `[HTTP Transport Security](http-transport-security.md)`

Reply:
Ah yes, perhaps you could do this instead?

```csharp
if (Uri.TryCreate(link, UriKind.Absolute, out Uri uri))
{
    return uri.Scheme switch
    {
        "http" or "https" or "ftp" => ...,
        "mailto" => ...,
        _ => ...
}
```